builtin/providers/google/resource_storage_bucket.go

```diff
@@ -89,6 +89,40 @@ func resourceStorageBucket() *schema.Resource {
 					},
 				},
 			},
+
+			"cors": &schema.Schema{
+				Type:     schema.TypeSet,
```

Comment:
No reason really - List semantics weren't important to it, and the first similar block I looked at was a Set. Changed to List for consistency with the rest of the file; it also ends up simplifying a type assertion that way.